README.md

```diff
@@ -1,6 +1,137 @@
 # stipulate
-A module extending the Fetch API with some useful default error handling and hooks.
+A module extending the Fetch API with some useful default error handling and data extraction.
 
 [![Build Status](https://travis-ci.org/yola/stipulate.svg?branch=master)](https://travis-ci.org/yola/stipulate)
+[![Coverage Status](https://coveralls.io/repos/github/yola/stipulate/badge.svg?branch=master)](https://coveralls.io/github/yola/stipulate?branch=master)
 
 Stipulate assumes the presence of a global `fetch`, in accordance with the [Fetch API spec](https://fetch.spec.whatwg.org/).
+
+## Usage
+
+```js
+import stipulate from 'stipulate';
+
+const options = {
```

Comment:
I would expect the first examples to be the most simple use case without options.  Something like:

```js
stipulate('http://www.mocky.io/v2/5710408d110000100a9e267a')
  .then(console.log);

// console:
// { connor: 'rocks' }


stipulate('http://www.mocky.io/v2/571040f0110000430a9e267b')
  .then(console.log)
  .catch(e => console.log('Fail: ' + e.response.status));

// console:
// Fail: 404
```